contrib/opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -128,95 +128,146 @@ def _transcribe_property(prop):
         return str(prop)
 
 
-def _find_timecode_mobs(item):
-    mobs = [item.mob]
-
-    for c in item.walk():
-        if isinstance(c, aaf2.components.SourceClip):
-            mob = c.mob
-            if mob:
-                mobs.append(mob)
-            else:
-                continue
+def _find_source_clip(item, op_group_found=None):
```

Comment:
From Zoom code review: we're wondering what `item` is passed in here?

Reply:
This was the segment of the 1st slot of the source clip from  _find_mob_chain_and_timecode, which might just be a SourceClip, but we might need to dig into the structure under it to find the SourceClip.

Reply:
This if/elif chain is attempting to handle the variety of AAFs in our test suite (both in OTIO, and from Matchbox) but we are still unclear of the full breadth of structures we might find within a SourceClip.